network/p2p/gossip/gossip.go

```diff
@@ -0,0 +1,141 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package gossip
+
+import (
+	"context"
+	"time"
+
+	"github.com/golang/protobuf/proto"
```

Comment:
This is deprecated I think

Reply:
Using `"google.golang.org/protobuf/proto"` now